networkx/algorithms/connectivity/cuts.py

```diff
@@ -21,6 +21,14 @@
 ]
 
 
+@nx._dispatch(
+    graphs={"G": 0, "auxiliary?": 4, "residual?": 5},
+    preserve_edge_attrs={
+        "auxiliary": {"capacity": None},
+        "residual": {"capacity": None},
```

Comment:
Should the default value for `"capacity"` edge attribute for connectivity and flow algorithms be 0?

Reply:
We decided that flow should have default 0 and capacity should have default 'inf'.